app/assets/stylesheets/custom.scss

```
@@ -1,9 +1,6 @@
 @import "bootstrap-sprockets";
```

trong css dùng nháy đơn đi e. Tương tự cho các chỗ khác nhé. :D
```suggestion
@import 'bootstrap-sprockets';
```